### PR TITLE
Init 1.0

### PR DESCRIPTION
**Pull request by sadoffToxir**
Добавил redux и и настроил init-1.0

### CODE REVIEW

**Comment by mtalstykh-exadel** [comment]
do not forget to use english

**Comment by sadoffToxir** [comment]
Настроил react-router-dom

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,27 @@
...
-        </a>
-      </header>
+      <Link to='/login'>Login</Link>
+      <Switch>
+        <Route path='/login' render={() => <Login/>}/>
+      </Switch>
     </div>
   );
 }
+//Позже будет вынесен в отдельный компонент
```
Comment (mtalstykh-exadel) [inline src/App.js:14]: and here too

Comment (sadoffToxir) [reply src/App.js:14]: ok

**Pull request closed without merging**